packages/codecs/ilp/src/utils/package.json

```diff
@@ -0,0 +1,6 @@
+{
+  "name": "@interledger/codecs-ilp/utils",
```

Comment:
This is an example of a way you can have `@interledger/x/y` imports work 
You'll note a symlink link from the packages/x to the y folder where this is kept ...
note the main/types/private

Reply:
I guess this would be for testing imports, or you could choose the `export namespace testing {...}` approach ... 

I'm loosely in favor of the latter

Reply:
I'm not sure if you actually require either if you don't care that the src is not importing from the build/ 

For tests is may not matter ... 

It may be about the IDE workings working better with the absolute imports ...

Reply:
I like the idea of naming the packages `@interledger/codecs/oer` if possible but I couldn't make that work, possibly because something else was broken

Reply:
The main challenge is getting the IDE to look at the src of dependencies so refactors and click-throughs work but not including them in the build.

When I was playing with paths I seemed to always end up with the compiled code of the dependencies included in the build folder.